src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -54,6 +52,28 @@ export interface SvgViewerProps {
   handleSearchChange?: () => void;
 }
 
+interface SvgViewerDocumentIdProps extends SvgViewerBasicProps {
+  // CDF fileId to fetch svg-document
+  documentId: number;
+}
+
+const instanceOfDocumentIdProps = (
+  object: any
+): object is SvgViewerDocumentIdProps => {
+  return !!object.documentId;
```

Comment:
you can create a utility
something like
```
const hasProperty = <PropertyType>(
  object: any,
  propertyName: string
): object is PropertyType => propertyName in object;
```

and then use it for `instanceOfDocumentIdProps ` and `instanceOfFileProps `

Reply:
@mrtraser suggestions?

Reply:
added